lib/gui/lvgl_common/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+set(ZEPHYR_CURRENT_LIBRARY lvgl)
```

Comment:
Problem is that `zephyr_library_named` contains an `add_library` and only one target can exist with the same name, if `zephyr_library_named(lvgl)` would be used you get following error:
```
  add_library cannot create target "lvgl" because another target with the
  same name already exists.  The existing target is an interface library
  created in source directory
  "/srv/workspace/dxplore/zephyr/code/zephyr/lib/gui/lvgl_v6".  See
  documentation for policy CMP0002 for more details.
```

A new library (eg. `lvgl_common`) could be created but as the code logicaly belongs to getter I opted for this approach.